project-production.yaml

```diff
@@ -0,0 +1,31 @@
+specVersion: 1.0.0
+name: turing-subql
+version: 0.0.4
+description: Any queries for subquery for Turing network
+repository: https://github.com/oak/oak-subql
+
+runner:
+  node:
+    name: "@subql/node"
+    version: "*"
+  query:
+    name: "@subql/query"
+    version: "*"
+
+schema:
+  file: ./schema.graphql
+network:
+  endpoint: 'wss://rpc.turing.oak.tech'
+  genesisHash: '0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d'
+  chaintypes:
+    file: ./dist/chaintypes.js
+dataSources:
+  - kind: substrate/Runtime
+    startBlock: 1
+    mapping:
+      file: ./dist/index.js
+      handlers:
+        - handler: handleBlock
+          kind: substrate/BlockHandler
+        - handler: handleEvent
```

Comment:
Why should we delete `eventHandlers.ts`?

I think it's an officially provided interface. should be kept.

And in this configuration file, `handleEvent` method is also configured.